packages/imperative/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Imperative package will be documented in this file.
 
 ## `8.10.4`
 
+- Enhancement: Added `DeferredPromise` class to Imperative to provide utilities for data synchronization. [#2405](https://github.com/zowe/zowe-cli/pull/2405)
```

Comment:
I forgot to mention that this should be moved into a new entry with the "recent changes" header